### PR TITLE
Fix flaky attribution test

### PR DESCRIPTION
**Pull request by evil159**
## Pull request checklist:
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add documentation comments for any added or updated public APIs.
 - [ ] Add any new public, top-level symbols to the Jazzy config's `custom_categories` (scripts/doc-generation/.jazzy.yaml)
 - [ ] Describe the changes in this PR, especially public API changes.
 - [ ] Add a changelog entry to to bottom of the relevant section (typically the `## main` heading near the top).
 - [ ] Update the guides (internal access only), README.md, and DEVELOPING.md if their contents are impacted by these changes.
 - [ ] Review and agree to the Contributor License Agreement ([CLA](https://github.com/mapbox/mapbox-maps-ios/blob/main/CONTRIBUTING.md#contributor-license-agreement)).

### CODE REVIEW

File: Tests/MapboxMapsTests/Style/AttributionTests.swift [hunk 1]
```diff
@@ -59,7 +59,9 @@ class AttributionTests: XCTestCase {
     }
 
     func testPlainTextAttributionParsing() throws {
-        let attributionString = String.randomASCII(withLength: 10)
+        var characterSet = CharacterSet(charactersIn: "©")
```
Comment (macdrevx) [inline Tests/MapboxMapsTests/Style/AttributionTests.swift:62]: Curious that randomASCII would emit a `©`, which is unicode `0xA9`. `randomASCII` generates characters in `0x20...0x7E`.

Comment (macdrevx) [reply Tests/MapboxMapsTests/Style/AttributionTests.swift:62]: Spaces are definitely possible (0x20)

**Review by macdrevx — APPROVED** [review]

File: Tests/MapboxMapsTests/Style/AttributionTests.swift (hunk 1 above)
Comment (evil159) [reply Tests/MapboxMapsTests/Style/AttributionTests.swift]: True that, removed it.

Comment (evil159) [reply Tests/MapboxMapsTests/Style/AttributionTests.swift]: True that, removed it.

**Pull request merged**